.github/workflows/system-tests.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 'DataDog/system-tests'
+          ref: 'piotr-wolski/update-node-js-test'
```

Comment:
This will need to be removed before merge

Reply:
Yes, it goes with this PR: https://github.com/DataDog/system-tests/pull/3346

Not sure in what order I should merge them (probably tests first)